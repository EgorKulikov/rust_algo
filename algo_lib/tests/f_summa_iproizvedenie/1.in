3
3
1 3 2
4
3 2
5 6
3 1
5 5
4
1 1 1 1
1
2 1
6
1 4 -2 3 3 3
3
2 -8
-1 -2
7 12
