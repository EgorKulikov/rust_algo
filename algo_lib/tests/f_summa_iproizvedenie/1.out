1 1 0 0
6
1 1 3
